packages/dashboard-core-plugins/src/panels/Panel.tsx

```diff
@@ -335,21 +346,23 @@ class Panel extends PureComponent<PanelProps, PanelState> {
       isRenamable,
     } = this.props;
     const { tab: glTab } = glContainer;
-    const { showRenameDialog, title } = this.state;
+    const { showRenameDialog, title, isWithinPanel } = this.state;
 
     return (
       <div
         className={classNames('h-100 w-100 iris-panel', className)}
         onFocusCapture={this.handleFocus}
         onBlurCapture={this.handleBlur}
+        ref={this.ref}
       >
         {children}
         <LoadingOverlay
           errorMessage={errorMessage}
           isLoaded={isLoaded}
           isLoading={isLoading}
         />
-        {glTab != null &&
+        {!isWithinPanel &&
+          glTab != null &&
```

Comment:
Right now we're defaulting to `isWithinPanel = true` on mount, and only setting it to false after the component is updated and we have a ref. I _believe_ that could leave a case where the portal here gets rendered, then we take it out immediately after on the next update. I'd flip the logic around so that we only render this after we verify that we are _not_ in another panel, just to be sure.

Reply:
That's why I default to true. Initially we're not sure if it's in another panel, so assume we are. Portal should not be rendered on first render until we determine if we're in a wrapped panel and then a rerender would trigger

Reply:
I feel like we're saying the same thing, but we both said it's defaulting to `true` which is incorrect - right now it's currently defaulting to `isWithinPanel: false` in the constructor. When it _should_ be `isWithinPanel: true`.

Reply:
🤦I should read my own code sometime instead of going off what I thought I wrote. Good catch